### DISCUSSION TITLE
Welcome to ArduinoStuff Discussions!

### DISCUSSION THREAD

**Discussion by The-Number1-Best-Coder** [Codinosaur/ArduinoStuff#6] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by The-Number1-Best-Coder** [comment]
The code is all well and fine
However the max brightness is quite low.

**Reply by The-Number1-Best-Coder** [reply]
I will check the max brightness with the resistor

**Comment by The-Number1-Best-Coder** [comment]
The problem was with the resistor, Everything is fine now.
The output in the serial monitor needs formatting and the output is'nt properly displayed in the plotter.
is due to it is a ```long```.

**Reply by AnYellowKing** [reply]
huh
it shouldnt be a long
